CppCoverage/DebugInformationEnumerator.cpp

```diff
@@ -301,6 +301,30 @@ namespace CppCoverage
 				THROW("DIA: Cannot find symbol");
 			}
 
+			// go from block to function as needed
+
+			DWORD tag{ 0 };
+			symbol->get_symTag(&tag);
+
+			CComPtr<IDiaSymbol> probe{ symbol };
+
+			while (tag != SymTagEnum::SymTagFunction)
```

Comment:
Or `tag == SymTagEnum::SymTagBlock` depending on what other possible values there are in these circumstances

Reply:
Could also pass the mixed-mode or not state here, and bypass this check in the pure-native case